config/settings.py

```diff
@@ -13,28 +16,33 @@
 
 ALLOWED_HOSTS = ["*"]
 
-
 # Application definition
-AUTHENTICATION_BACKENDS = ("app.backends.CaseInsensitiveModelBackend",)
+AUTHENTICATION_BACKENDS = ("users.backends.CaseInsensitiveModelBackend",)
 
 INSTALLED_APPS = [
-    "app.apps.AppConfig",
     "django.contrib.admin",
     "django.contrib.auth",
     "django.contrib.contenttypes",
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "django.contrib.humanize",
+    "django.contrib.sites",
+
     "mptt",
     "rest_framework",
     "rest_framework.authtoken",
     "rest_auth",
-    "django.contrib.sites",
+
     "allauth",
     "allauth.account",
     "allauth.socialaccount",
     "rest_auth.registration",
+
+    "entries.apps.EntriesConfig",
+    'messeges.apps.MessegesConfig',
```

Comment:
Typo `messeges` -> `messages`

Reply:
you cant use this because django already use it Messages App

Reply:
Well then you have to come up with a new name :D 
Maybe `private_messages`?

Reply:
I prefer not to use the _ to create Django applications. We can change the model name to Message

Reply:
Model name is good, I'm not fine with a typo in apps name. If not `private_messages` then maybe `inbox`, `private`, `conversations`, etc.